### ISSUE TITLE
Why does `Psbt` not include por commitment (proof of reserves) 

### ISSUE THREAD

**Issue by tcharding** [rust-bitcoin/rust-bitcoin#3423]
BIP-174 includes a keytype in the input map `PSBT_IN_POR_COMMITMENT ` but we don't include it, was there a reason for this. Its a UTF-8 encode string (easy for us, just a `String`). 

Seems trivial to add so I'm wondering if there was a reason it was left out?

Thanks

**Comment by DanGould** [comment]
The bip was probably updated to include that field after being implemented here

**Comment by apoelstra** [comment]
Yes, this was added as part of BIP 127 in https://github.com/bitcoin/bips/pull/756 in Nov 2018.

Meanwhile our PSBT stuff was implemented here in #103 in Aug 2018.

We can go ahead and add it.

**Comment by tcharding** [comment]
Note that adding this requires adding serde migration code.

**Comment by apoelstra** [comment]
If the current serde serialization can't handle us adding new field names, we definitely need to replace it with the standard `base64` serialization and be done with it. (But yes, this will require migration code.)
